server/api/reports.py

```diff
@@ -742,6 +743,9 @@ def sampled_batch_rows(election: Election, jurisdiction: Jurisdiction = None):
 @api.route("/election/<election_id>/report", methods=["GET"])
 @restrict_access([UserType.AUDIT_ADMIN])
 def audit_admin_audit_report(election: Election):
+    if len(list(election.rounds)) == 0:
```

Comment:
is election.rounds the current round of an audit we are on or something?

Reply:
`election.rounds` is all of the rounds of the audit. so if it's empty, that means the audit hasn't started yet